helm-chart/splunk-operator/crds/enterprise.splunk.com_clustermanagers.yaml

```diff
@@ -63,63 +61,32 @@ spec:
                 description: Mock to differentiate between UTs and actual reconcile
                 type: boolean
               affinity:
-                description: Kubernetes Affinity rules that control how pods are assigned
-                  to particular nodes.
+                description: Kubernetes Affinity rules that control how pods are assigned to particular nodes.
                 properties:
                   nodeAffinity:
-                    description: Describes node affinity scheduling rules for the
-                      pod.
+                    description: Describes node affinity scheduling rules for the pod.
                     properties:
                       preferredDuringSchedulingIgnoredDuringExecution:
-                        description: The scheduler will prefer to schedule pods to
-                          nodes that satisfy the affinity expressions specified by
-                          this field, but it may choose a node that violates one or
-                          more of the expressions. The node that is most preferred
-                          is the one with the greatest sum of weights, i.e. for each
-                          node that meets all of the scheduling requirements (resource
-                          request, requiredDuringScheduling affinity expressions,
-                          etc.), compute a sum by iterating through the elements of
-                          this field and adding "weight" to the sum if the node matches
-                          the corresponding matchExpressions; the node(s) with the
-                          highest sum are the most preferred.
+                        description: The scheduler will prefer to schedule pods to nodes that satisfy the affinity expressions specified by this field, but it may choose a node that violates one or more of the expressions. The node that is most preferred is the one with the greatest sum of weights, i.e. for each node that meets all of the scheduling requirements (resource request, requiredDuringScheduling affinity expressions, etc.), compute a sum by iterating through the elements of this field and adding "weight" to the sum if the node matches the corresponding matchExpressions; the node(s) with the highest sum are the most preferred.
```

Comment:
any reason why this line is showing up like this? did we use a different version of jq? Probably the only change. in this file should should be from v3->v4?

Reply:
these are new one generated from develop branch. I do not wanted to keep the old ones its always better to copy from latest develop branch. so i prefer if we keep these ones. formatting should  not make much difference.